flink-connectors/flink-connector-aws-kinesis-firehose/src/test/java/org/apache/flink/connector/firehose/sink/KinesisFirehoseSinkWriterTest.java

```diff
@@ -101,6 +101,5 @@ void getNumRecordsOutErrorsCounterRecordsCorrectNumberOfFailures()
                 .withMessageContaining(
                         "Unable to execute HTTP request: Connection refused: localhost/127.0.0.1:443");
         assertThat(ctx.metricGroup().getNumRecordsOutErrorsCounter().getCount()).isEqualTo(12);
-        assertThat(ctx.metricGroup().getNumRecordsSendErrorsCounter().getCount()).isEqualTo(12);
```

Comment:
Shall we keep the check? IIUC, the result of send error counter should be equal to out error counter.